x-pack/plugins/security/public/management/users/user_utils.ts

```diff
@@ -4,6 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
 import { User } from '../../../common/model';
 
 export const isUserReserved = (user: User) => user.metadata?._reserved ?? false;
+
+export const isUserDeprecated = (user: User) => user.metadata?._deprecated ?? false;
+
+export const getExtendedUserDeprecationNotice = (user: User) => {
+  const reason = user.metadata?._deprecated_reason ?? '';
+  return i18n.translate('xpack.security.management.users.extendedUserDeprecationNotice', {
+    defaultMessage: `The {username} user is deprecated. {reason}`,
```

Comment:
Perhaps this should change so it will be consistent with the reason?

```suggestion
    defaultMessage: `The [{username}] user is deprecated. {reason}`,
```

Reply:
The current text is consistent with the deprecation message we display for roles today. Gail had actually requested that we remove the brackets from the reason when we did the role deprecations, but then that change would have been inconsistent with the rest of the messages in ES. Overall, I'm 🤷‍♂️ about it, I could go either way.

Given my beautiful backstory, which would you prefer? I'll defer to you 😄